eskip/template_test.go

```diff
@@ -253,6 +253,16 @@ func TestTemplateApplyContext(t *testing.T) {
 		"hello ",
 		false, // path is empty
 
+	}, {
+		"request rawquery",
+		"hello ${request.rawquery}",
+		&filtertest.Context{
+			FRequest: &http.Request{
+				URL: parseUrl("http://example.com/path?q-Q=foo&P_p=bar"),
```

Comment:
Please add a test case for:
- url without query `http://example.com/foo`
- url with a query without value `http://example.com/foo?bar`

Reply:
Maybe also a case for URL-encoded value

Reply:
I extended the test case `request rawquery` accordingly and added an additional test case `request rawquery is empty`.
Please have a look.